cmd/process-agent/collector.go

```diff
@@ -305,9 +305,12 @@ func (l *Collector) consumePayloads(results *api.WeightedQueue, fwd forwarder.Fo
 		}
 		result := item.(*checkResult)
 		for _, payload := range result.payloads {
-			forwarderPayload := forwarder.Payloads{&payload.body}
-			var responses chan forwarder.Response
-			var err error
+			var (
+				forwarderPayload = forwarder.Payloads{&payload.body}
+				responses        chan forwarder.Response
+				err              error
+				updateRTStatus   = true
```

Comment:
nit: it's up to you but I feel like it's better to have default to `false` and only set to `true` in `case checks.RTProcess.Name()`?

Reply:
Actually looking at it again, not only orchestrator check would reset it, network check could also do it, so I feel like what I suggested above is needed?